OpenXmlPowerTools/PegBase.cs

```diff
@@ -1643,71 +2116,107 @@ public bool Into(Matcher toMatch,out string into)
                 return false;
             }
         }
-        public bool Into(Matcher toMatch,out PegBegEnd begEnd)
+
+        public bool Into(Matcher toMatch, out PegBegEnd begEnd)
         {
             begEnd.posBeg_ = pos_;
-            bool bMatches = toMatch();
+            var bMatches = toMatch();
             begEnd.posEnd_ = pos_;
             return bMatches;
         }
-        public bool Into(Matcher toMatch,out int into)
+
+        public bool Into(Matcher toMatch, out int into)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename the parameter 'into' so that it does not duplicate the method name.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)